lib/fluent/compat/output.rb

```diff
@@ -111,6 +111,10 @@ module ChunkSizeCompatMixin
       def size
         self.bytesize
       end
+
+      def contains_size
```

Comment:
`contains_size` looks a method to return boolean for me.
How about `size_of_events` or so?